config/deploy/dev.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'server_settings'
+
+server 'sdr-services-dev.stanford.edu', user: 'sdr2service', roles: %w{app}
+set :deploy_to, "/var/sdr2service/#{ENV['SDR_APP']}"
```

Comment:
I think that would be a good follow on change. However, at this juncture I'm not sure what the valid values for `ENV['SDR_APP']` are.